### PR TITLE
Update CI yamls

### PR DESCRIPTION
**Pull request by jeefberkey**

### CODE REVIEW

**Comment by jeefberkey** [comment]
:white_check_mark: https://gitlab.com/simp/pupmod-simp-pki/pipelines/17804666

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+    key: "${CI_PROJECT_NAMESPACE}__bundler"
     paths:
       - '.vendor'
       - 'vendor'
 
 .setup_bundler_env: &setup_bundler_env
   before_script:
-    - gem install bundler
+    - '(find .vendor | wc -l) || :'
+    - bundle || gem install bundler --no-rdoc --no-ri
```
Comment (lnemsick-simp) [inline .gitlab-ci.yml]: 'bundle' -> 'bundle check' ?

File: Gemfile [hunk 2]
```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'puppet-lint-empty_string-check',   :require => false
   gem 'puppet-lint-trailing_comma-check', :require => false
   gem 'simp-rspec-puppet-facts', ENV.fetch('SIMP_RSPEC_PUPPET_FACTS_VERSION', '~> 2.0')
-  gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', '~> 4.0')
+  gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', '~> 5.2')
```
Comment (lnemsick-simp) [inline Gemfile]: gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', ['>= 5.2.0', '< 6.0'])

File: .gitlab-ci.yml (hunk 1 above)
Comment (lnemsick-simp) [reply .gitlab-ci.yml]: disregard

File: Gemfile (hunk 2 above)
Comment (lnemsick-simp) [reply Gemfile]: disregard.

**Pull request merged**